CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 
 ## Unreleased
 
+### Fixes
```

Comment:
Ah didn't notice the changelog was wrong on main

Reply:
yeah, probably could even be when i merged another pr 😅. Not sure, though